# Generated by Django 3.0.2 on 2020-06-16 13:38

import datetime
from django.db import migrations, models
import django.db.models.deletion
from django.utils.timezone import utc


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0010_auto_20200616_1319'),
    ]

    operations = [
        migrations.AlterField(
            model_name='complain',
            name='time',
            field=models.DateTimeField(default=datetime.datetime(2020, 6, 16, 13, 38, 55, 394197, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='doctor',
            name='time',
            field=models.DateTimeField(default=datetime.datetime(2020, 6, 16, 13, 38, 55, 393611, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='order',
            name='time',
            field=models.DateTimeField(default=datetime.datetime(2020, 6, 16, 13, 38, 55, 391841, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='storerestro',
            name='time',
            field=models.DateTimeField(default=datetime.datetime(2020, 6, 16, 13, 38, 55, 392934, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='tax',
            name='time',
            field=models.DateTimeField(default=datetime.datetime(2020, 6, 16, 13, 38, 55, 394795, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='userdetails',
            name='serves',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to='api.cat'),
        ),
        migrations.AlterField(
            model_name='userdetails',
            name='time',
            field=models.DateTimeField(default=datetime.datetime(2020, 6, 16, 13, 38, 55, 389763, tzinfo=utc)),
        ),
    ]
